Composer/packages/client/src/components/ProjectTree/treeItem.tsx

```diff
@@ -94,6 +107,7 @@ export const TreeItem: React.FC<ITreeItemProps> = props => {
         data-testid={`DialogTreeItem${link.id}`}
         onRenderItem={onRenderItem}
         onRenderOverflowButton={onRenderOverflowButton(link.isRoot, isActive)}
+        doNotContainWithinFocusZone={true}
```

Comment:
We should make a note (here in a comment, maybe) to remove this prop once we upgrade to Fluent 8.0 (whenever that happens); this behavior is going to be the default from then on.